app/assets/stylesheets/locals.css.scss

```diff
@@ -51,6 +51,12 @@ $green: #0eda83;
   text-decoration: underline;
 }
 
+.sidebar-link-2 {
```

Comment:
**suggestion:**

1. Can you use [Sass' `@extend` rule](https://sass-lang.com/documentation/at-rules/extend) to share these CSS rules with `.sidebar-link`?
2. Please use a more descriptive class name for this CSS rule.